pkg/assets/apps/bindata.go

```diff
@@ -863,7 +863,7 @@ spec:
       serviceAccountName: kubevirt-hostpath-provisioner-admin
       containers:
         - name: kubevirt-hostpath-provisioner
-          image: quay.io/kubevirt/hostpath-provisioner
+          image: quay.io/kubevirt/hostpath-provisioner:v0.8.0
```

Comment:
don't edit this file. It is generated from https://github.com/redhat-et/microshift/blob/main/assets/apps/000_80_hostpath-provisioner-daemonset.yaml by https://github.com/redhat-et/microshift/blob/main/scripts/bindata.sh